docs/changelog.rst

```diff
@@ -1105,7 +1105,7 @@ New Functionality
 .. code-block:: python
 
   engine = GlobusComputeEngine(
-        address="127.0.0.1",
+        address="localhost",
         heartbeat_period_s=1,
```

Comment:
This is changing history; please don't.  Instead, we can mention the change to the default config in a new change log fragment.